Client/Frontend/Browser/Onboarding/Welcome/OnboardingAdblockDisconnect.swift

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Shared
+
+private let log = Logger.browserLogger
+
+struct OnboardingDisconnectItem: Codable {
+    let properties: [String]
+    let resources: [String]
+    
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        properties = try container.decode([String].self, forKey: .properties)
+        resources = try container.decode([String].self, forKey: .resources)
+    }
```

Comment:
Can be removed

Reply:
Removed at https://github.com/brave/brave-ios/pull/4508/commits/a51ac89b29bda9b22be0eed68d06ded22e4dfad0